scripts/generate_export.py

```diff
@@ -32,7 +33,7 @@ def assign_confidences(results, model_name):
     return preds_final
 
 
-def get_page_extractions(indico_wrapper, submission, model_name, post_review=False):
+def get_page_extractions(indico_wrapper, submission, model_name, post_review=True):
```

Comment:
No reason to change the default argument here. This behavior is controlled by the `post_review` variable ~ line 314.

Reply:
I may have accidentally changed it. Please ignore this.